src/main/resources/db.changeLog/changeLog-master.xml

```diff
@@ -2,7 +2,14 @@
 <databaseChangeLog
         xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+        xmlns:pro="http://www.liquibase.org/xml/ns/pro"
         xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
-      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.1.xsd">
-    <includeAll path="com/example/changelog/"/>
-</databaseChangeLog>
\ No newline at end of file
+		http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd
+		http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd
+		http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-latest.xsd">
+
+    <include file="db.changeLog/changeLog-1.xml" relativeToChangelogFile="true"/>
```

Comment:
application does not start, because `changeLog-master.xml` has 2 errors.

you can add temprorary `spring.liquibase.drop-first=true` property to `application.properties` to test your liquibase. this property will clear all tables and data from database. remove property when you finish testing